lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -53,86 +79,142 @@ class ArgumentsForwarding < Base
 
         minimum_target_ruby_version 2.7
 
-        MSG = 'Use arguments forwarding.'
-
-        # @!method use_rest_arguments?(node)
-        def_node_matcher :use_rest_arguments?, <<~PATTERN
-          (args ({restarg kwrestarg} $_) $...)
-        PATTERN
-
-        # @!method only_rest_arguments?(node, name)
-        def_node_matcher :only_rest_arguments?, <<~PATTERN
-          {
-            (send _ _ (splat (lvar %1)))
-            (send _ _ (hash (kwsplat (lvar %1))))
-          }
-        PATTERN
-
-        # @!method forwarding_method_arguments?(node, rest_name, block_name, kwargs_name)
-        def_node_matcher :forwarding_method_arguments?, <<~PATTERN
-          {
-            (send _ _
-              (splat (lvar %1))
-              (block-pass {(lvar %2) nil?}))
-            (send _ _
-              (splat (lvar %1))
-              (hash (kwsplat (lvar %3)))
-              (block-pass {(lvar %2) nil?}))
-          }
-        PATTERN
+        FORWARDING_LVAR_TYPES = %i[splat kwsplat block_pass].freeze
+
+        FORWARDING_MSG = 'Use shorthand syntax `...` for arguments forwarding.'
+        ARGS_MSG = 'Use anonymous positional arguments forwarding: `*`.'
```

Comment:
I think it'd be better to use parens here instead of `: *` - (`*`). To me it reads a bit better. Not an expert on the subject though. :-)

Reply:
Sure thing, I've pushed another a commit 👍

Reply:
I think the build failure is unrelated to this PR, and unfortunately I can't rerun it